### PR TITLE
Fix: Update dependencies to fix vulnerabilities

### LINKED ISSUE #4594: [Bug] added 69 vulnerabilities (62 moderate, 7 high) to my project
# 🐞 Bug report

## Description
installing this module as a dev dependency added 69 vulnerabilities (62 moderate, 7 high) to project

bug: this project relies has direct dependencies containing security risks
expected outcome: remove/upgrade this projects dependency revealed via `npm audit`

## Details

1. [clone the nodeproto project](https://github.com/noahehall/nodeproto/)
2. run `npm i -D hint`
3. run `npm audit`

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #4594

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

**Comment by antross** [comment]
@sarvaje looks like the updates just inched `extension-browser` over the size limit:
> Bundle size of 732KB exceeds limit of 730KB

I say go ahead and raise the limit to 735KB for now. Also, you'll probably want to check `utils-worker` to see if it was similarly impacted.

**Comment by sarvaje** [comment]
@antross `utils-worker` looks good :)

**Comment by antross** [comment]
Was `packages/hint-no-vulnerable-javascript-libraries/src/snyk-snapshot.json` intended to be deleted in this PR?

**Comment by sarvaje** [comment]
@antross I have replaced `cpx` in favor of  `copyfiles`. This change makes `yarn audit` to report 0 vulnerabilites in my machine.

**Comment by sarvaje** [comment]
>Was `packages/hint-no-vulnerable-javascript-libraries/src/snyk-snapshot.json` intended to be deleted in this PR?

no.

**Comment by sarvaje** [comment]
`packages/hint-no-vulnerable-javascript-libraries/src/snyk-snapshot.json` wasn't deleted, it is the minified version I think. I will revert that.

File: packages/extension-browser/package.json [hunk 1]
```diff
@@ -140,7 +140,7 @@
     "lint:dependencies": "node ../../scripts/lint-dependencies.js",
     "lint:md": "node ../../scripts/lint-markdown.js",
     "prebuild": "npm-run-all prebuild:*",
-    "prebuild:assets": "cpx \"./src/**/*.{html,json,png}\" dist/bundle && cpx \"./src/*.svg\" dist/bundle && cpx \"./src/browser-action/*.svg\" dist/bundle/browser-action",
+    "prebuild:assets": "copyfiles \"./src/**/*.{html,json,png}\" dist/bundle && copyfiles \"./src/*.svg\" dist/bundle && copyfiles \"./src/browser-action/*.svg\" dist/bundle/browser-action",
```
Comment (antross) [inline packages/extension-browser/package.json:143]: Judging from the CI failures, this `copyfiles` command isn't quite working the same as `cpx`. We're not getting `_locales/en/messages.json` copied over.

Comment (sarvaje) [reply packages/extension-browser/package.json]: yep, I was working on the fix. Please, take a look to my last commit.

File: packages/extension-browser/package.json [hunk 2]
```diff
@@ -140,7 +140,7 @@
     "lint:dependencies": "node ../../scripts/lint-dependencies.js",
     "lint:md": "node ../../scripts/lint-markdown.js",
     "prebuild": "npm-run-all prebuild:*",
-    "prebuild:assets": "copyfiles \"./src/**/*.{html,json,png}\" dist/bundle && copyfiles \"./src/*.svg\" dist/bundle && copyfiles \"./src/browser-action/*.svg\" dist/bundle/browser-action",
+    "prebuild:assets": "copyfiles -u 1 \"./src/**/*.{html,json,png}\" dist/bundle && copyfiles -f \"./src/*.svg\" dist/bundle && copyfiles -f \"./src/browser-action/*.svg\" dist/bundle/browser-action",
```
Comment (Jessen-Bau) [inline packages/extension-browser/package.json:143]: Sehr schön.

**Pull request merged**